developer/src/tike/main/Keyman.Developer.System.TikeMultiProcess.pas

```diff
@@ -67,17 +74,34 @@ destructor TTikeMultiProcess.Destroy;
   inherited Destroy;
 end;
 
+type
+  TSortByProjectFilename = class(TComparer<TTikeProcess>)
+    function Compare(const Left, Right: TTikeProcess): Integer; override;
+  end;
+
 function TTikeMultiProcess.Enumerate: TTikeProcessList;
 var
   tp: TTikeProcess;
   p: TMultiProcessInstance;
   j: TJSONValue;
   jo: TJSONObject;
+  SortByProjectFilename: TSortByProjectFilename;
 begin
   Result := TObjectList<TTikeProcess>.Create;
   MultiProcessCoordinator.Enumerate;
   for p in MultiProcessCoordinator.Processes do
   begin
+    if p.Identifier = '*' then
+    begin
+      tp := TTikeProcess.Create;
+      tp.IsProjectOpen := False;
+      tp.IsTemporaryProject := False;
+      tp.ProjectFilename := '';
```

Comment:
Why is 
`tp.IsTemporaryProject := False` when l. 22 compares

```
tp.IsTemporaryProject := tp.ProjectFilename = '';
```

Would it be easier to manage `IsTemporaryProject` as (psuedocode)

```
function IsTemporaryProject: Boolean;
begin
  Result := ProjectFilename = '';
end;
```

Reply:
That doesn't quite work -- it is a separate flag. This would work though:

```
function IsTemporaryProject: Boolean;
begin
  Result := IsProjectOpen and (ProjectFilename = '');
end;
```